transmogrifier/sources/xml/marc.py

```diff
@@ -961,6 +957,15 @@ def get_numbering(cls, source_record: Tag) -> str | None:
             return " ".join(numbering_values) or None
         return None
 
+    @classmethod
+    def get_physical_description(cls, source_record: Tag) -> str | None:
+        if physical_description_values := [
+            cls.create_subfield_value_string_from_datafield(datafield, "abcefg", " ")
+            for datafield in source_record.find_all("datafield", tag="300")
+        ]:
+            return " ".join(physical_description_values) or None
+        return None
+
```

Comment:
This pattern shows up enough that we might consider abstracting it out further for easier re-use

Reply:
Abstracted it has!